### ISSUE TITLE
模型训练很慢是什么问题呀？

### ISSUE THREAD

**Issue by yumeng88** [yeyupiaoling/PP-YOLOE#8]

**Comment by yumeng88** [comment]
训练集图片36000张   1920*1080  三个种类   目标框的数量在200000个左右  硬件设备是一块1080TI

**Comment by yumeng88** [comment]
[07/12 22:31:55] ppdet.engine INFO: Epoch: [0] [ 340/2311] learning_rate: 0.000425 loss_xy: 5.585110 loss_wh: 8.053473 loss_iou: 40.807060 loss_obj: 36.960297 loss_cls: 11.582250 loss: 106.673569 eta: 54 days, 5:33:41 batch_cost: 2.9940 data_cost: 2.7134 ips: 5.3440 images/s
[07/12 22:33:03] ppdet.engine INFO: Epoch: [0] [ 360/2311] learning_rate: 0.000450 loss_xy: 5.039430 loss_wh: 8.437138 loss_iou: 37.917873 loss_obj: 32.460560 loss_cls: 10.454338 loss: 95.006454 eta: 54 days, 9:49:05 batch_cost: 3.2062 data_cost: 2.9550 ips: 4.9903 images/s
[07/12 22:34:10] ppdet.engine INFO: Epoch: [0] [ 380/2311] learning_rate: 0.000475 loss_xy: 5.357884 loss_wh: 8.278685 loss_iou: 40.448494 loss_obj: 36.259827 loss_cls: 10.271514 loss: 99.963409 eta: 54 days, 12:01:29 batch_cost: 3.1353 data_cost: 2.8720 ips: 5.1032 images/s
[07/12 22:35:14] ppdet.engine INFO: Epoch: [0] [ 400/2311] learning_rate: 0.000500 loss_xy: 5.314033 loss_wh: 7.778206 loss_iou: 39.397888 loss_obj: 36.490501 loss_cls: 9.850763 loss: 98.626900 eta: 54 days, 10:39:34 batch_cost: 2.9791 data_cost: 2.7004 ips: 5.3708 images/s

**Comment by yumeng88** [comment]
设置成100epoch还会有7、8天

**Comment by yeyupiaoling** [comment]
@yumeng88 我看你训练日志，一秒才训练5张图片，是有点慢，我的是60张

你有没有修改输入大小的配置？

你看下你GPU的使用率，看是不是因为数据读取导致训练变慢的。你用了多少个线程读取数据？

**Comment by yumeng88** [comment]
@yeyupiaoling GPU使用率很低，具体的线程读取我没关注，我重新训练再看一下

**Comment by yeyupiaoling** [comment]
这应该是数据读取慢导致的，数据读取慢有很多原因，例如磁盘的读写速度很慢，又或者CPU预处理很慢等、
